cost-analyzer/values.yaml

```diff
@@ -512,8 +512,9 @@ kubecostModel:
 
   # The name of the Secret containing a bucket config for ETL backup.
   # etlBucketConfigSecret:
-  # The name of the Secret containing a bucket config for Federated storage.
-  # federatedStorageConfigSecret:
+  # The name of the Secret containing a bucket config for Federated storage. The contents should be stored
+  # under a key named federated-store.yaml.
+  federatedStorageConfigSecret: ""
```

Comment:
Why uncomment this? I think we only need it for certain configurations

Reply:
Mistake and fixed! I appreciate you, Michael.